portfolio/src/main/webapp/projects.html

```diff
@@ -8,7 +8,12 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script> 
     <script> 
       $(function(){
-        $("#navbar-content").load("navbar.html"); 
+        $("#navbar-content").load("navbar.html", function() {
+          let currPage = $(this)[0].ownerDocument.location.pathname == '/' ? '/index.html' : $(this)[0].ownerDocument.location.pathname;
+          currPage = currPage.slice(1, -5);
+          const currPageLink = document.getElementById(currPage);
+          $('#' + currPage).addClass('nav-item active');
+        }); 
```

Comment:
I think in this case if we're doing this on a per-page basis, the callback can just be
function(){
    $("#projects").addClass('nav-item active');
} 
as we know the id we want on each page. I know its a bit more "hard coded" than the current page way, but my argument for that in this case is if we happen to add a new page, we're going to have to add a callback to it anyway so we might as well make it simpler to read :D